app/views/layouts/application.html.erb

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>Humble Hearts</title>
-  <%= stylesheet_link_tag    'application', media: 'all' %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-	<%= render '/layouts/header' %>
-  <div id="wrapper">
-	  <p class="notice"><%= notice %></p>
-	  <p class="alert"><%= alert %></p>
-	  <p class="notice"><%= notice %></p>
-	  <p class="alert"><%= alert %></p>
-	 <%= yield %>
-  </div>
-	<%= render '/layouts/footer' %>
-</body>
-</html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title><%= content_for?(:title) ? yield(:title) : "foundation-rails" %></title>
+
+    <%= stylesheet_link_tag    "application" media: 'all' %>
+    <%= javascript_include_tag "vendor/modernizr" %>
+    <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body>
+		<%= render '/layouts/header' %>
+		 <div id="wrapper">
+		  <p class="notice"><%= notice %></p>
```

Comment:
Foundation doesn't require turbolinks, I'm pretty sure. What changes need to be made before I implement styling with foundation?